resources/svc/default-mysql.yml

```diff
@@ -1,18 +1,16 @@
-version: '3.1'
+# This configuration is intended for development purpose, it's **your** responsibility to harden it for production
+version: '3.8'
 
 services:
   mysql:
-    image: mysql
-    restart: always
+    image: mysql:8.0.26
```

Comment:
Hi @jeffgo10 , The image version could be moved to package.json and added here as variable. @cecchisandrone WDYT?

Reply:
We may do this to have the same approach as keycloak. In any case this is not a custom image.

Reply:
@alepintus i'm up for it. should we implement this @cecchisandrone ?

Reply:
@jeff keep it as it is. @alepintus will do the version centralization in the scope of ENG-3672